packages/material-ui/src/Input/Input.js

```diff
@@ -111,6 +111,8 @@ const Input = React.forwardRef(function Input(inProps, ref) {
   const props = useThemeProps({ props: inProps, name: 'MuiInput' });
   const {
     disableUnderline,
+    components = {},
+    componentsProps: componentsPropsInput,
```

Comment:
Usually, we add the Prop suffix when we need to rename an input prop. I know it's a bit weird with the existing props suffix

```suggestion
    componentsProps: componentsPropsProp,
```

Reply:
Alright, let's keep it consistent :)